lib/create-plugin.js

```diff
@@ -195,23 +195,15 @@ function createPlugin(instrumentationApi, config = {}) {
           let transactionName = '*'
           let segmentName = UNKNOWN_OPERATION
 
-          // always use context.source so we can get both queries and persisted queries
-          // see: https://github.com/apollographql/apollo-server/blob/2bccec2c5f5adaaf785f13ab98b6e52e22d5b22e/packages/apollo-server-core/src/requestPipeline.ts#L232
-          let query = responseContext.source
-
-          if (query) {
```

Comment:
do we need tests to verify a query isn't added as an attr when it cannot parse query?

Reply:
Yes, I'll add one.